pkgs/by-name/vi/virt-v2v/package.nix

```diff
@@ -0,0 +1,108 @@
+{
+  stdenv,
+  lib,
+  testers,
+  fetchurl,
+  pkg-config,
+  makeWrapper,
+  autoreconfHook,
+  bash-completion,
+  OVMF,
+  qemu,
+  ocamlPackages,
+  perl,
+  cpio,
+  getopt,
+  libosinfo,
+  pcre2,
+  libxml2,
+  jansson,
+  glib,
+  libguestfs-with-appliance,
+  cdrkit,
+  nbdkit,
+  withWindowsGuestSupport ? true,
+  pkgsCross, # for rsrvany
+  virtio-win,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "virt-v2v";
+  version = "2.6.0";
+
+  src = fetchurl {
+    url = "https://download.libguestfs.org/virt-v2v/${lib.versions.majorMinor finalAttrs.version}-stable/virt-v2v-${finalAttrs.version}.tar.gz";
+    sha256 = "sha256-W7t/n1QO9UebyH85abtnSY5i7kH/6h8JIAlFQoD1vkU=";
+  };
+
+  prePatch = ''
+    patchShebangs .
```

Comment:
Nit: prefer to patch shebangs in postPatch over prePatch, so that `patches` can apply cleanly to the pristine source. (I know it's unlikely to be an issue in practice.)

Data point:

```
$ git grep -B1 patchShebangs | grep prePatch | wc -l
27
$ git grep -B1 patchShebangs | grep postPatch | wc -l
592
```